<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="pbxcore"
           tests="23"
           failures="2"
           errors="0"
           skipped="0"
           time="932.202">
  <properties />
            <testcase name="B2_AttendTranferTimeout" time="44.573" classname="com.yeastar.swebtest.testcase.RegressionCase.pbxcase.Extension">
                <failure type="java.lang.AssertionError"
                  message="预计产生多条CDR，但没有找到匹配"
              >
<![CDATA[java.lang.AssertionError: 预计产生多条CDR，但没有找到匹配
  com.yeastar.swebtest.tools.ysassert.YsAssert.fail(YsAssert.java:1092)
  com.yeastar.swebtest.driver.YSMethod.YS_Extension.checkCDR(YS_Extension.java:348)
  com.yeastar.swebtest.testcase.RegressionCase.pbxcase.Extension.B2_AttendTranferTimeout(Extension.java:219)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
  java.lang.reflect.Method.invoke(Method.java:498)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
  org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
  org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
  org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
  org.testng.TestRunner.privateRun(TestRunner.java:774)
  org.testng.TestRunner.run(TestRunner.java:624)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
  org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
  org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
  org.testng.SuiteRunner.run(SuiteRunner.java:261)
  org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
  org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
  org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
  org.testng.TestNG.run(TestNG.java:1048)
  org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:293)
  org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
  org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:91)
  org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:200)
  org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153)
  org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
]]>
      </failure>

          </testcase>
            <testcase name="A1_sip" time="41.174" classname="com.yeastar.swebtest.testcase.RegressionCase.pbxcase.Extension">
                <failure type="java.lang.AssertionError"
                  message=" 录音失败"
              >
<![CDATA[java.lang.AssertionError:  录音失败
  com.yeastar.swebtest.tools.ysassert.YsAssert.fail(YsAssert.java:1092)
  com.yeastar.swebtest.testcase.RegressionCase.pbxcase.Extension.A1_sip(Extension.java:136)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
  java.lang.reflect.Method.invoke(Method.java:498)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
  org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
  org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
  org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
  org.testng.TestRunner.privateRun(TestRunner.java:774)
  org.testng.TestRunner.run(TestRunner.java:624)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
  org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
  org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
  org.testng.SuiteRunner.run(SuiteRunner.java:261)
  org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
  org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
  org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
  org.testng.TestNG.run(TestNG.java:1048)
  org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:293)
  org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
  org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:91)
  org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:200)
  org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153)
  org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
]]>
      </failure>

          </testcase>
  
  
            <testcase name="D1_callforward" time="32.805" classname="com.yeastar.swebtest.testcase.RegressionCase.pbxcase.Extension" />
                <testcase name="H1_resetToDefault" time="7.094" classname="com.yeastar.swebtest.testcase.RegressionCase.pbxcase.Extension" />
                <testcase name="A4_fxs" time="40.288" classname="com.yeastar.swebtest.testcase.RegressionCase.pbxcase.Extension" />
                <testcase name="D2_cancellForward" time="31.901" classname="com.yeastar.swebtest.testcase.RegressionCase.pbxcase.Extension" />
                <testcase name="K1_check_Voicemail" time="55.786" classname="com.yeastar.swebtest.testcase.RegressionCase.pbxcase.Extension" />
                <testcase name="A2_sip" time="29.861" classname="com.yeastar.swebtest.testcase.RegressionCase.pbxcase.Extension" />
                <testcase name="I_Voicemail" time="79.770" classname="com.yeastar.swebtest.testcase.RegressionCase.pbxcase.Extension" />
                <testcase name="E1_whenbusy" time="48.844" classname="com.yeastar.swebtest.testcase.RegressionCase.pbxcase.Extension" />
                <testcase name="C2_callpark" time="53.139" classname="com.yeastar.swebtest.testcase.RegressionCase.pbxcase.Extension" />
                <testcase name="A3_fxs" time="35.342" classname="com.yeastar.swebtest.testcase.RegressionCase.pbxcase.Extension" />
                <testcase name="A0_Register" time="29.046" classname="com.yeastar.swebtest.testcase.RegressionCase.pbxcase.Extension" />
                <testcase name="F2_NoAnswer" time="55.853" classname="com.yeastar.swebtest.testcase.RegressionCase.pbxcase.Extension" />
                <testcase name="C1_callpark" time="50.221" classname="com.yeastar.swebtest.testcase.RegressionCase.pbxcase.Extension" />
                <testcase name="G1_dnd" time="13.208" classname="com.yeastar.swebtest.testcase.RegressionCase.pbxcase.Extension" />
                <testcase name="G2_canceldnd" time="30.970" classname="com.yeastar.swebtest.testcase.RegressionCase.pbxcase.Extension" />
                <testcase name="B3_BlindTransfer" time="40.817" classname="com.yeastar.swebtest.testcase.RegressionCase.pbxcase.Extension" />
                <testcase name="F1_NoAnswer" time="67.158" classname="com.yeastar.swebtest.testcase.RegressionCase.pbxcase.Extension" />
                <testcase name="E2_cancelWhenBusy" time="40.273" classname="com.yeastar.swebtest.testcase.RegressionCase.pbxcase.Extension" />
                <testcase name="J_internal" time="48.435" classname="com.yeastar.swebtest.testcase.RegressionCase.pbxcase.Extension" />
                <testcase name="K2_Check_OneTouchRecord" time="4.840" classname="com.yeastar.swebtest.testcase.RegressionCase.pbxcase.Extension" />
                <testcase name="B1_AttendTranfer" time="50.804" classname="com.yeastar.swebtest.testcase.RegressionCase.pbxcase.Extension" />
      
</testsuite>
